abouts/govbuy.md

```diff
@@ -1,14 +1,59 @@
 # רכש פתוח
-## האינקוויזיציה הספרדית {team}
+## מהו אתר "רכש פתוח"? {govbuy}
+אתר "רכש פתוח" כולל נתונים על תהליכי רכש והתקשרויות ממשלתיות עם ספקים. המערכת כוללת מידע על מכרזים מרכזיים, מכרזים משרדיים, פטור ממכרז והתקשרויות. האתר נועד לאפשר ולהקל על בעלי עסקים להצטרף למעגל הספקים הממשלתי ולעובדי ממשלה הפועלים בתחומי הרשת לייעל ולשפר את קבלת ההחלטות בנוגע לניצול כספי הציבור.
 
-לאינקוויזיציה הספרדית היו מספר שיטות פעולה:
+## מאיפה הנתונים? {data-sources}
+### אתר מנהל הרכש המרכזי
+נתונים בנוגע לתהליכי רכש ואישורם (מכרזים מרכזיים, מכרזים משרדיים ופטור ממכרז) לפני ביצוע ההתקשרות בפועל, מתבסס על הנתונים המוצגים ב<a href="https://www.mr.gov.il/About/Pages/default.aspx" target="_blank">אתר הרכש הממשלתי</a>.מינהל הרכש הממשלתי הינו גוף מטה בחטיבת נכסים רכש ולוגיסטיקה באגף החשב הכללי. תפקידו לבצע ולהתוות את מדיניות הרכש והלוגיסטיקה הממשלתית. מינהל הרכש מתמחה בביצוע רכש מרכזי, בגיבוש מדיניות לביצוע רכש במגזר הממשלתי ובסיוע לגופים ממשלתיים בביצוע ויישום מדיניות הרכש במשרדם.
```

Comment:
את הלינקים עדיף לכתוב בסגנון markdown ולהימנע ככל האפשר מהטמל

```
[link title](http://where/to/link/to)
```

[link title](http://example.com)

Reply:
@akariv how can I use markdown to open link in a new / separate tab?

Reply:
Yes, it's a configuration parameter for the markdown converter.

Reply:
not sure I understand - should I include it as a parameter or it will be added automatically by the converter for every external link (asking because on my local it was opening in the same tab when I tried, don't mind trying again.)